mne/epochs.py

```diff
@@ -818,6 +869,59 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         return self._evoked_from_epoch_data(data, self.info, picks, n_events,
                                             kind, self._name)
 
+    def regress(self, on, fit_intercept=True, by=None):
```

Comment:
pinging @agramfort on this one...he wrote it into the prototype that we put together and nobody else complained about it when we showed folks on the gitter so I wrote it into here as well. I do see @kingjr's point...WDYT alex?